src/lib.rs

```diff
@@ -64,7 +64,6 @@ mod node;
 mod tree;
 
 pub use build::*;
-pub use macros::*;
```

Comment:
this is because of
```
error: unused import: `macros::*`
  --> src/lib.rs:67:9
   |
67 | pub use macros::*;
   |         ^^^^^^^^^
   |
note: the lint level is defined here
  --> src/lib.rs:59:9
   |
59 | #![deny(warnings)]
   |         ^^^^^^^^
   = note: `#[deny(unused_imports)]` implied by `#[deny(warnings)]`
```